components/gamification/store/StoreItemCard.tsx

```diff
@@ -1,24 +1,67 @@
-import React from 'react';
-import { View, Text, Pressable, StyleSheet } from 'react-native';
+import React, { useRef, useEffect } from 'react';
+import {View, Text, Pressable, StyleSheet, Animated, Easing} from 'react-native';
 import { Ionicons } from '@expo/vector-icons';
 import Colors from '../../../theme/colors';
 import { IStoreItemWithQuantity } from '../types';
-import { TYPE_DOUBLE_ANSWER, TYPE_ELIMINATE_WRONG_ANSWER } from './constants';
+import {
+  TYPE_DOUBLE_ANSWER,
+  TYPE_DOUBLE_ANSWER_OLD,
```

Comment:
I know @can-aslan didn't do this but in general, let's not put "TYPE" into our type namings. Types & variables are already distinguishable with their use cases. This makes it harder to read.